Data-tranformations/fairgenomes/samples.yarrml.yaml

```diff
@@ -0,0 +1,22 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
+  OBI: http://purl.obolibrary.org/obo/OBI_
+  EFO: http://www.ebi.ac.uk/efo/EFO_ # I added this one
+  GSSO: http://purl.obolibrary.org/obo/GSSO_ # I added this one
+
+mappings:
+  fairgenomes:
+    sources:
+      - [samples.csv~csv] # I am going to name it like this samples.csv
+    s: ega.dataset:$(accession_id) # I am unsure for this part, I have not changed
+    po:
+      - [a, NCIT:C43376]
+      - [OMIABIS:0000006, ega.dataset:$(accession_id)~iri] #I am unsure for this part,too
```

Comment:
I’m also unsure about this. The definition of the predicate here is:
> Class: sample collection or study identifier
> Term IRI: http://purl.obolibrary.org/obo/OMIABIS_0000006
> Definition: A sample collection or study proper name that is centrally registered and unique within one framework.

I guess we could see the analysis as a group of samples and then it does make sense. Otherwise, I guess we could use the class that OMIABIS has narrowed down the definition of (i.e. the “superclass”):
> Class: identifier
> Term IRI: http://purl.obolibrary.org/obo/IAO_0020000
> Definition: An information content entity that is the outcome of a dubbing process and is used to refer to one instance of entity shared by a group of people to refer to that individual entity.

Reply:
Now I am a bit confused. If we say that we have the `samples` mapping. In the `prefixes` I am going to change: ega.dataset: https://www.ebi.ac.uk/ega/datasets/ to ega.samples: https://metadata.ega-archive.org/samples/ ... but then in the `s: ega.dataset:$(accession_id)` do we continue to have the ega.dataset? and then after the `po` is again the ega.datasets. It feels strange. Am I going to keep in the `prefixes` the  ega.datasets: https://metadata.ega-archive.org/datasets/ and add the ega.samples, too? Or do we have to replace the ega.datasets with the ega.samples. We must clarify that before I make any changes.

Reply:
I guess from the next messages that you wrote that I have to replace ega.dataset with ega.samples ...